api/app/hfi/__init__.py

```diff
@@ -1 +1,2 @@
 """ Module for HFI calculations """
+from .hfi_calc import hydrate_fire_centres, calculate_latest_hfi_results
```

Comment:
Do we need from at all if it's relative?

Reply:
Oh hey - sorry - missed your comment - but yeah - you still need to say where you're importing from a.f.a.i.k.